RetailSuiteAccounting/UserInterface/01a_Bookings.md

```diff
@@ -0,0 +1,424 @@
+# Postings
+
+*Accounting > Post > Tab POSTINGS*
+
+  ![Post](../../Assets/Screenshots/RetailSuiteAccounting/Book/Bookings/Bookings.png "[Post]")
+
+The *POSTINGS* tab displays all existing postings in the selected posting period in chronological order. The posting details displayed in columns are read-only.
+
+- *Seq. no.*  
+This column displays the sequential number.
+
+  > [Info] The posting numbers must be sequential and are added by the system automatically.
+
+- *BP*  
+This column displays the posting period (month/journal).
+
+- *Turnover*  
+This column displays the turnover value.
+
+- *Contra account*  
+This column displays the contra account number.
+
+- *Receipt1*  
+This column displays the receipt 1 value, usually the invoice number.
+
+  > [Info] The information provided in the *Receipt 1* field is essential, as the system identifies and groups together any subsequent postings taking this field as reference, for example for an invoice settlement or a credit note in case of refund.
```

Comment:
```suggestion
Receipt 1 value, usually the invoice number. The information provided in this field is essential, as the system identifies and groups together any subsequent postings taking this field as reference, for example for an invoice settlement or a credit note in case of refund.
```